x-pack/plugins/ml/server/models/model_management/memory_usage.ts

```diff
@@ -199,6 +199,8 @@ export class MemoryUsageService {
               ...rest,
               ...modelSizeState,
               node: nodeRest,
+              // @ts-ignore
+              key: `${rest.deployment_id}_${nodeId}`,
```

Comment:
Can the `TODO set node_id to ml:true when elasticsearch client is updated.` be updated yet?  

Should the `NodesOverviewResponse` interface be updated to include `key`?

Reply:
Updated in https://github.com/elastic/kibana/pull/155375/commits/c076dfef65ea1d39272f407e2711252094882683